main.cpp

```diff
@@ -119,6 +119,17 @@ int main() {
       shaka::Symbol("/"),
       create_node(div_numbers));
 
```

Comment:
Technically, these changes were not a part of the pull request implementing GCNode. This inclusion was part of your sprint 1 task assignment to build familiarity with the source code. While this isn't technically a big deal, as you haven't really broken anything in the source code by creating this closure and binding it in the top level environment, from a pedagogical perspective, changes to main.cpp should be segregated to their own PR, separate from your GCNode implementation. I'll defer to @VermillionAzure as to whether we should have you do a `git checkout main.cpp` from upstream master and then push this version of the file to your origin GcNode branch.